### ISSUE TITLE
mybinder.org not working

### ISSUE THREAD

**Issue by wstomv** [gibiansky/IHaskell#910]
When I try the [mybinder.org](https://mybinder.org/v2/gh/gibiansky/IHaskell/master) link, it takes a while to load things, and then I can upload a Haskell notebook or create a new Haskell notebook. But in either case, the IHaskell kernel keeps on dying, or the notebook simply fails to connect to the kernel. I have not been able to evaluate even a single Haskell code cell.

The session is still alive (reloading the dashboard shows the file listing), and it shows the restarting kernels (under the Running tab).

Any idea what is the matter?

**Comment by vaibhavsagar** [comment]
I'm seeing this too. I'll have to build the Docker image locally so I can look at the log output and try to figure out what's going wrong.

**Comment by vaibhavsagar** [comment]
Okay, what seems to be happening is that IHaskell is calling `stack exec env` to support `stack` and that is installing the latest LTS which uses a newer version of GHC than the one in `stack.yaml`, and that is causing issues.

```bash
[vaibhavsagar@nixos:~/repos/IHaskell]$ docker run -it ihaskell:latest /bin/bash
jovyan@f66717f4e8cb:~$ stack exec env
Writing implicit global project config file to: /home/jovyan/.stack/global-project/stack.yaml
Note: You can change the snapshot via the resolver field there.
Using latest snapshot resolver: lts-12.7
Downloaded lts-12.7 build plan.    
Preparing to install GHC to an isolated location.
This will not interfere with any system-level installation.
ghc-8.4.3: download has begun
```

**Comment by vaibhavsagar** [comment]
As a workaround, opening a new IHaskell notebook from within the `ihaskell` directory seems to work correctly.

**Comment by wstomv** [comment]
@vaibhavsagar Thanks. That works; also when I upload a Haskell notebook to that directory.

**Comment by vaibhavsagar** [comment]
Closed by #952.
